CMakeLists.txt

```diff
@@ -11,12 +11,15 @@ add_subdirectory(third_party/pybind11)
 set(TENSEAL_BASEDIR tenseal)
 set(SOURCES
     ${TENSEAL_BASEDIR}/binding.cpp
+    ${TENSEAL_BASEDIR}/sealapi.cpp
```

Comment:
Not necessary for now, but this line should be made optional, so that we would compile seal_api into TenSEAL or not.